### DISCUSSION TITLE
does this support paid news platform subscriptions?

### DISCUSSION THREAD

**Discussion by reallymemorable** [FreshRSS/FreshRSS#5411] [Q&A] [answered]
e.g. if i am a paid subscriber to Bloomberg.com or NYTimes.com, can i somehow link that membership to this RSS reader so that i can see articles from that source?

**Comment by Frenzie** [comment]
That depends on if a feed protected by HTTP username and password is part
of that subscription.

**Reply by reallymemorable** [reply]
don't some RSS readers allow you to pass your credentials?

**Reply by aledeg** [reply]
As @Frenzie mentioned, if the credentials for your feeds are through HTTP authentication, you can add them in your feeds configuration. They will be used for each connection on those feeds.

**Reply by reallymemorable** [reply]
does this library support it? 

if so, where do i provide the credentials in the code?

**Reply by Frenzie** [reply, marked as answer]
While adding a feed you have to expand "advanced".
![image](https://github.com/FreshRSS/FreshRSS/assets/202757/1ceae54b-2759-4294-abda-65c76358f26c)
